rust/processor/src/utils/util.rs

```diff
@@ -450,6 +450,15 @@ pub struct AggregatorSnapshotU64 {
     pub value: BigDecimal,
 }
 
+// TODO: How is this different from AgggregatorU64?
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct AggregatorU128 {
```

Comment:
if it's not diff from U64 can you just rename and reuse? These names don't matter. E.g. you can name it AggregatorBigint or something.